docs/cli/edgedb_project/edgedb_project_info.rst

```diff
@@ -18,6 +18,11 @@ Description
 This command provides information about the project instance, such as
 name and the project path.
 
+.. note::
+
+    The ``edgedb project info`` command is not intended for use with production
```

Comment:
If you do link your project to a remote instance, this one is supported.
```bash
$ edgedb project info
┌───────────────┬───────────────────────────────────────┐
│ Instance name │ do-test                               │
│ Project root  │ /home/fmoor/src/sample-remote-project │
└───────────────┴───────────────────────────────────────┘
```

Reply:
Removed this one too.